pkg/nodes/queue.go

```diff
@@ -0,0 +1,108 @@
+package nodes
+
+import (
+	"context"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/celestiaorg/torch/config"
+	"github.com/celestiaorg/torch/pkg/db/redis"
+	"github.com/celestiaorg/torch/pkg/metrics"
+)
+
+var (
+	taskQueue     = make(chan config.Peer) // taskQueue channel for pending tasks (peers to process later).
```

Comment:
it's not good to do taskQueue like this. 

You can either create a struct that holds the queue functions or only create it in AddPhase